Casks/thumbsup.rb

```diff
@@ -4,9 +4,13 @@
 
   url "https://s3.amazonaws.com/DTWebsiteSupport/download/freeware/thumbsup/#{version}/ThumbsUp.app.zip",
       verified: "s3.amazonaws.com/DTWebsiteSupport/"
-  appcast "https://www.macupdater.net/cgi-bin/extract_text/extract_text_split_easy.cgi?url=https://www.devontechnologies.com/support/download&splitter_1=ThumbsUp&index_1=1&splitter_2=disabled&index_2=0"
   name "ThumbsUp"
   homepage "https://www.devontechnologies.com/apps/freeware"
 
+  livecheck do
+    url "https://www.devontechnologies.com/support/download"
+    regex(%r{<td>ThumbsUp</td>\s*<td>(\d+(?:\.\d+)+)</td>}i)
```

Comment:
```suggestion
    url :homepage
    regex(%r{href=.*?/thumbsup/v?(\d+(?:\.\d+)+)/ThumbsUp\.app\.zip}i)
```

I'm not quite sure why the download page doesn't contain a link to the zip file when fetching the page with livecheck or `curl`. It appears in the source when fetching with a browser, even with JavaScript disabled. I tried setting a user agent string in `curl` and it still doesn't appear in the source. Rather strange.

Regardless, the homepage contains a link to the file that's used as the cask `url`, so it would be preferable to check that anyway.